api/src/test/resources/application.properties

```diff
@@ -0,0 +1,23 @@
+# H2 in-memory database configuration
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+spring.datasource.platform=h2
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
+
+# Hibernate settings
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
+
+
+# jwt config
+jwt.secret=scary_secret
```

Comment:
Why this? I believe there @gebz97 had a script to generate a secret right? Use that script, and reference the secret from the .env file

Reply:
It's tests. Do we need to keep env variables in tests? What for?